Formula/lima.rb

```diff
@@ -0,0 +1,25 @@
+class Lima < Formula
+  desc "Linux virtual machines"
+  homepage "https://github.com/AkihiroSuda/lima"
+  url "https://github.com/AkihiroSuda/lima/archive/v0.3.0.tar.gz"
+  sha256 "0debed5cd2fcb34ce90114c4f07a3610a3b2d4aaf4592608135e9053cb5ce213"
```

Comment:
v0.4.0 is now available: https://github.com/AkihiroSuda/lima/releases/tag/v0.4.0

```
  url "https://github.com/AkihiroSuda/lima/archive/v0.4.0.tar.gz"
  sha256 "482e4cff30103aac5bf93feb76721f1ada26a7618e503365040c4adae2a28043"
```